lib/dao.js

```diff
@@ -2669,6 +2669,19 @@ DataAccessObject.updateAll = function(where, data, options, cb) {
   this.applyScope(query);
   this.applyProperties(data);
 
+  var doValidate = undefined;
+  if (options.validate === undefined) {
+    if (Model.settings.validateUpdate === undefined) {
```

Comment:
I know they are aliases. It's just that when I see `update` in code, I have to always remember that it's an alias for `updateAll`. I am trying to say that the name `update` is not clear enough, while `updateAll` makes the intent much more clear.